### DISCUSSION TITLE
JAR support within Ory Hydra

### DISCUSSION THREAD

**Discussion by mchristophe** [ory/hydra#3305] [Q&A] [answered]
Hello,

As the title mentions, is there any ongoing work on JAR support within Ory Hydra?
This became a standard as found here: https://datatracker.ietf.org/doc/html/rfc9101

This together with PAR would represent a high security improvement in your great great work! I am using Ory Hydra as OAuth/OID provider in the financial industry, with online payment :)

Thanks for the heads up!

Matheu

**Comment by vinckr** [comment, marked as answer]
Hello @mchristophe 
It has been a while, forgive the late answer. 

As of now there is no ongoing work towards JAR support.  
However you can contact the Ory team if you are interested in [support for on-premise deployments of Ory Hydra](https://www.ory.sh/support/). Also new features can be talked about, it always depends on customer interest so [feel free to contact the team](https://www.ory.sh/contact/) for a short chat if you are interested.

**Reply by vinckr** [reply]
See this comment relating to PAR: https://github.com/ory/fosite/pull/660#issuecomment-1188958354
